asistente_ladm_col/gui/qgis_model_baker/dlg_import_schema.py

```diff
@@ -472,6 +400,7 @@ def on_process_finished(self, exit_code, result):
 
         self.txtStdout.setTextColor(QColor(color))
         self.txtStdout.append(message)
+        QCoreApplication.processEvents()
```

Comment:
Why this one? Since we are done, the loop will return the control to the GUI anyways.